bin/test-example

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
It works on the machines of mine where `/bin/sh` isn't bash, so I have changed all shebangs to `/bin/sh`. I would like to note for the record that it will remain important to me that these scripts do not use bash-specific features, maybe I should document that somewhere (don't know where to...)